lib/PrimitiveHelpers.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+var Cesium = require('cesium');
+var deepEqual = require('deep-equal');
+
+var Cartesian3 = Cesium.Cartesian3;
+var Matrix4 = Cesium.Matrix4;
+var defined = Cesium.defined;
+
+var AccessorReader = require('./AccessorReader');
+var readAccessor = require('./readAccessor');
+var writeAccessor = require('./writeAccessor');
+
+module.exports = {
```

Comment:
@lilleyse @lasalvavida can you two please review the design of this class?  I don't think it is needed.  Some functions could be private in the files they are used in; others could be standalone functions.

Let me know if you need help.